code/modules/gear_presets/uscm_ship.dm

```diff
@@ -252,16 +252,9 @@
 	if (new_human.client && new_human.client.prefs && (new_human.client.prefs.backbag == 1))
 		back_item = /obj/item/storage/backpack/marine/tech
 
-	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/almayer/mt(new_human), WEAR_L_EAR)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/marine/officer/engi(new_human), WEAR_BODY)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/marine(new_human), WEAR_FEET)
-	new_human.equip_to_slot_or_del(new /obj/item/clothing/gloves/yellow(new_human), WEAR_HANDS)
-	new_human.equip_to_slot_or_del(new /obj/item/clothing/head/welding(new_human), WEAR_HEAD)
-	new_human.equip_to_slot_or_del(new /obj/item/storage/belt/utility/full(new_human), WEAR_WAIST)
 	new_human.equip_to_slot_or_del(new back_item(new_human), WEAR_BACK)
-	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/general/medium(new_human), WEAR_R_STORE)
-	new_human.equip_to_slot_or_del(new /obj/item/device/demo_scanner(new_human), WEAR_L_STORE)
-	new_human.equip_to_slot_or_del(new /obj/item/storage/bag/trash(new_human), WEAR_L_HAND)
 	new_human.equip_to_slot_or_del(new /obj/item/storage/backpack/marine/engineerpack/welder_chestrig, (new_human), WEAR_R_HAND)
```

Comment:
While you're here, this chestrig spawn is currently bugged.
There's a rogue comma in the proc arguments causing `(new_human)` to be sent as the `slot`, which then causes the chestrig to be deleted because that's invalid.

Up to you if you want to fix or remove this, since the PR is changing around MT gear anyway.

Reply:
i could take a look tomorrow in 10+ hour...

Reply:
removed it from being spawn there now it's a choice in the vendor.
didn't found the proc you where talking about...

Reply:
This is the rouge comma in the proc arguments: 
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/b8898e68-707d-43b6-b971-5a2baa47a9c8)